rskj-core/src/main/java/co/rsk/peg/federation/FederationChangeResponseCode.java

```diff
@@ -8,6 +8,7 @@ public enum FederationChangeResponseCode {
     PROPOSED_FEDERATION_ALREADY_EXISTS(-4),
     PENDING_FEDERATION_NON_EXISTENT(-1),
     RETIRING_FEDERATION_NON_EXISTENT(-1),
+    PROPOSED_FEDERATION_NON_EXISTENT(-1),
```

Comment:
```suggestion
    FEDERATION_NON_EXISTENT(-1),
```
What do you think? We wrap it in a single response code

Reply:
Sounds good, updated all references